PubMatic/src/main/java/com/applovin/mediation/adapters/PubMaticMediationAdapter.java

```diff
@@ -509,5 +543,155 @@ public void onAdClicked(@NonNull final POBBannerView view)
         }
     }
 
+    private class NativeAdLoaderListener implements POBNativeAdLoaderListener
+    {
+
+        private final MaxNativeAdAdapterListener maxNativeAdAdapterListener;
+        NativeAdLoaderListener(MaxNativeAdAdapterListener maxNativeAdAdapterListener)
+        {
+            this.maxNativeAdAdapterListener = maxNativeAdAdapterListener;
+        }
+
+        @Override
+        public void onAdReceived(@NonNull POBNativeAdLoader pobNativeAdLoader, @NonNull POBNativeAd pobNativeAd)
+        {
+            log( "Native ad loaded" );
+
+            nativeAd = pobNativeAd;
+            final MaxNativeAd.Builder builder = new MaxNativeAd.Builder();
+            builder.setAdFormat( MaxAdFormat.NATIVE );
+
+            // Map title
+            POBNativeAdTitleResponseAsset title = pobNativeAd.getTitle();
+            if(title != null)
```

Comment:
Same here and elsewhere in this file, please add a space after `if` to keep the code consistency. 

```suggestion
            if (title != null)
```